Source/Core/Core/TitleDatabase.cpp

```diff
@@ -89,6 +89,16 @@ static bool IsWiiTitle(const std::string& game_id)
   return !IsGCTitle(game_id);
 }
 
+static bool IsJapaneseGCTitle(const std::string& game_id)
+{
+  return IsGCTitle(game_id) && DiscIO::RegionSwitchGC(game_id[3]) == DiscIO::Region::NTSC_J;
+}
+
+static bool IsNonJapaneseGCTitle(const std::string& game_id)
+{
+  return IsGCTitle(game_id) && DiscIO::RegionSwitchGC(game_id[3]) != DiscIO::Region::NTSC_J;
```

Comment:
I'm fine with keeping it that way, fwiw, since the function is not really that long.